src/Files.Uwp/ViewModels/ToolbarViewModel.cs

```diff
@@ -915,6 +915,11 @@ public async Task SetPathBoxDropDownFlyoutAsync(MenuFlyout flyout, PathBoxItem p
 
         public async Task CheckPathInput(string currentInput, string currentSelectedPath, IShellPage shellPage)
         {
+            if (currentInput.Contains("/"))
+            {
+                currentInput = currentInput.Replace("/", "\\", StringComparison.Ordinal);
```

Comment:
Could you check if this creates issues when browsing an FTP server?

Reply:
That’s a good thought. I didn’t consider that. I’ll get back to you.

Reply:
Yeah it creates issues for ftp paths. Maybe I'll close this one for now and have another look.

Reply:
Thanks, can't test right know but commit 944596c looks good.